Clinical-Genomics/development
# Add requirements.txt

How to test?

```
git clone -b <branch name> https://github.com/Clinical-Genomics/development.git
cd development
conda create -n D_mkdocs
source activate D_mkdocs
pip install -r requirements.txt
mkdocs serve
```
mkdocs should serve just fine and not complain about missing plugins
